login/build.gradle

```diff
@@ -0,0 +1,101 @@
+apply plugin: 'com.android.library'
```

Comment:
as most of the feature module gradle files will contain the same dependencies, you can create another gradle file like common.gradle, and it will include all that is on this file, then on this file you just need to put this line `apply from: "$rootProject.projectDir/common.gradle"` , and if you need a dependency that will not be shared, you can add a dependency block also on this file and put it on it

Reply:
common example https://gist.github.com/oscarg798/699c6a1c11534272f10146283bed85e1

Reply:
feature one https://gist.github.com/oscarg798/def23a5347bf19792e3ff6ce1353b67e